src/modules/services/components/ClientServicesPage.tsx

```diff
@@ -40,6 +42,13 @@ const columns: ColumnDef<ServiceType>[] = [
     optional: true,
```

Comment:
Should Service Details now be removed? https://docs.google.com/spreadsheets/d/17ZpdB0fH6kUL14KDRWrlukwIH93BNx9grrNZApLpUwg/edit?disco=AAABeWYJLxU